### ISSUE TITLE
HTML autocomplete on Known User Info (Sign Up & Login Forms)

### ISSUE THREAD

**Issue by bpmcneilly** [internetarchive/openlibrary#5003]
WCAG 2.1 added the success criterion [1.3.5 Identify Input Purpose](https://www.w3.org/WAI/WCAG21/Understanding/identify-input-purpose) which, effectively requires the use of the html `autocomplete` attribute on certain form fields.

The [full list of fields requiring this attribute](https://www.w3.org/TR/WCAG21/#input-purposes) is available in the WCAG docs & the [html spec describing the attribute](https://html.spec.whatwg.org/#autofilling-form-controls:-the-autocomplete-attribute), but the only instance of values that seem relevant are in our sign up or log in flows. I think these are limited to:

- email
- username
- password
    - There are separate values for an existing vs new password

Most of the other captured items (e.g. credit card info, birthdates, addresses) we don't collect, so we don't need to include them. I don't think this type of information is captured anywhere else in the site, but let me know if I'm missing anything!

### Describe the problem that you'd like solved
Basically, we just need to find the relevant templates & add in the ``autocomplete`` attribute & the relevant values. An example for the final markup would be:
```
<form id="register" class="login olform" name="register" method="post">
    <div class="formElement">
        <div class="label">
          <label for="username">Email</label> ...</div>
        <div class="input">
          <input type="email" class="required" id="username" name="username" value="" placeholder="Email" autocomplete="email" ...>
          <span class="invalid clearfix" htmlfor="username"></span>
        </div>
    </div>
    <div class="formElement">
        <div class="label"><label for="password">Password</label> ...</div>
        <div class="input">
            <input type="password" class="required" placeholder="Password" id="password" name="password" autocomplete="current-password" ...>
            <span class="invalid clearfix" htmlfor="password"></span>
        </div>
    </div>
    ...
</form>
```

### Proposal & Constraints
There shouldn't be any blockers or issues with this. In addition to being a WCAG requirement, it will help users sign up for an account & log in faster and more accurately!

### Additional context
Relvant files:

- openlibrary/openlibrary/templates/account/create.html
- openlibrary/openlibrary/templates/login.html

### Stakeholders
@mekarpeles - just to confirm I'm not missing some part of openlibrary that collects this type of user info

**Comment by Priyanshujha11** [comment]
Can you assign this to me

**Comment by Yashs911** [comment]
@bpmcneilly I think we also need to add (https://openlibrary.org/contact?path=/help)

- Name
- Email

**Comment by bpmcneilly** [comment]
@Priyanshujha11 - how's this going? Got any questions of issues with it on your end?

Also, good spot @Yashs911 - we should definitely include those two in any upcoming PR

**Comment by Sadashii** [comment]
@lokesh I'd like to work on this issue
